[Discounts] Discount policies (#2771)
ibexa/documentation-developer: docs/permissions/policies.md

| <nobr>`commerce`</nobr> | <nobr>`currency`</nobr> | manage currencies |
|                         | <nobr>`region`</nobr>   | manage regions    |

#### Discounts [[% include 'snippets/lts-update_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]

The discount policies decide which actions can be executed by given user or user group.

!!! caution "Customers and discount policies"

>     Customers do not need any policies to use the discounts on the [storefront](storefront.md).

Reviewer: ```suggestion
    Customers don't need any policies to use the discounts on the [storefront](storefront.md).
```
